File: stress/t/ctl_minmaxmed.f
      PROGRAM CTLMMM
      INTEGER*8 A,B,C,MN,MX,MED,SUM
      A=17
      B=4
      C=9
      IF (A.LT.B) THEN
        MN=A
      ELSE
        MN=B
      END IF
      IF (C.LT.MN) THEN
        MN=C
      END IF
      IF (A.GT.B) THEN
        MX=A
      ELSE
        MX=B
      END IF
      IF (C.GT.MX) THEN
        MX=C
      END IF
      SUM=A+B+C
      MED=SUM-MN-MX
      WRITE(*,'(A,I0)') 'mn=',MN
      WRITE(*,'(A,I0)') 'mx=',MX
      WRITE(*,'(A,I0)') 'med=',MED
      END

File: stress/t/ctl_minmaxmed.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t a=17,b=4,c=9,mn=0,mx=0,med=0,sum=0;
  if(a<b){ mn=a; } else { mn=b; }
  if(c<mn){ mn=c; } else { mn=mn; }
  if(a>b){ mx=a; } else { mx=b; }
  if(c>mx){ mx=c; } else { mx=mx; }
  sum=a+b+c;
  med=sum-mn-mx;
  printf("mn="); printf("%ld",(long)mn); printf("\n");
  printf("mx="); printf("%ld",(long)mx); printf("\n");
  printf("med="); printf("%ld",(long)med); printf("\n");
  return 0;
}
